test/e2e/framework/network/utils.go

```diff
@@ -117,6 +124,9 @@ type NetworkingTestConfig struct {
 	HostTestContainerPod *v1.Pod
 	// if the HostTestContainerPod is running with HostNetwork=true.
 	HostNetwork bool
+	// if the test pods are listening on sctp port. We need this as sctp tests
+	// are marked as disruptive as they may load the sctp module.
+	SctpEnabled bool
```

Comment:
`SCTPEnabled`. (In golang style, acronyms in names are always either all-caps or all-lowercase, never just initial-caps.)